examples/webgl_nearestneighbour.html

```diff
@@ -182,51 +187,58 @@
 				controls.handleResize();
 			}
 
-			function animate() {
+			function animate () {
```

Comment:
Thanks for the code cleanup, but according to [mrdoobapproves](http://zz85.github.io/mrdoobapproves/) and [this comment](https://github.com/zz85/mrdoobapproves/issues/45#issuecomment-131428034), the whitespace pattern should be one of:
```
function animate() {

}

var animate = function () {

};
```